src/main/java/net/snowflake/client/core/SFSession.java

```diff
@@ -483,7 +483,12 @@ public synchronized void open() throws SFException, SnowflakeSQLException {
         .setApplication((String) connectionPropertiesMap.get(SFSessionProperty.APPLICATION))
         .setServiceName(getServiceName())
         .setOCSPMode(getOCSPMode())
-        .setHttpClientSettingsKey(httpClientSettingsKey);
+        .setHttpClientSettingsKey(httpClientSettingsKey)
+        .setDisableConsoleLogin(
+            connectionPropertiesMap.get(SFSessionProperty.DISABLE_CONSOLE_LOGIN) != null
+                ? getBooleanValue(
+                    connectionPropertiesMap.get(SFSessionProperty.DISABLE_CONSOLE_LOGIN))
+                : true);
```

Comment:
can we create a package scoped const `DISABLE_CONSOLE_LOGIN_DEFAULT_VALUE` and use it also in `SFLoginInput?`

Reply:
All constant have same scope as earlier one.

Reply:
I am asking for it to have only only one place where we define default value for this parameter. It would be completely new constant

Reply:
Will cleanup later as part of other constant.